.github/workflows/mirror-images.yml

```diff
@@ -12,9 +12,19 @@ on:
         type: string
         required: true
         default: "1.0.0"
+  workflow_run:
+    workflows: ['Create Production Tag']
+    types: [completed]
+#    branches: [main]
+
 jobs:
   mirror:
     runs-on: ubuntu-20.04
+    env:
+      ECR_REPOSITORY: "dsva/next-build-node"
+      IMAGE_TAG: ${{ inputs.image_tag }}
+    if: ${{ !(github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'failure') }}
```

Comment:
If the event name is 'workflow_run' AND the result was 'failure', do not run the job

if the event name is 'workflow_run' AND the result was 'success', or if the event name is anything else, go ahead with the job.

I had to think about it for a while too.

Reply:
> Is this saying the mirror images workflow only runs if the Create Production Tag succeeds?

Effectively yes; or, if it's triggered manually it will also run.

Reply:
copy, thanks for clarifying